modules/search/client/config/search.client.routes.js

```diff
@@ -80,6 +80,15 @@
         data: {
           pageTitle: 'Search'
         }
+      }).
+      state('search-users', {
```

Comment:
@simison  I find this search-users /search/members inconsistent. Shall we move everything to members naming convention?